src/test/java/com/rakovets/course/java/core/practice/string/StringUtilTest.java

```diff
@@ -0,0 +1,289 @@
+package com.rakovets.course.java.core.practice.string;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class StringUtilTest {
+    static Stream<Arguments> concatProviderArguments() {
+        return Stream.of(
+                Arguments.of("HelloJava", "Hello", "Java"),
+                Arguments.of("Java", "", "Java"),
+                Arguments.of("Hello", "Hello", "")
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("concatProviderArguments")
+    void Glue(String expectedString, String str1, String str2) {
```

Comment:
"Violation: 'nameMethodTest' was not used for test-method naming
Solution: Rename method
"